msupply-foundation/open-msupply
# 3939 decimal quantities requisition line

Fixes #3939 

# 👩🏻‍💻 What does this PR do?

Updates requisition line quantities to be float

# 🧪 Testing
- Run tests
- Create internal order in msupply containing float quantities, the line should get synced
- go through UI and look for related problems
